src/Hpack/Defaults.hs

```diff
@@ -33,8 +33,8 @@ defaultsCachePath :: FilePath -> Github -> FilePath
 defaultsCachePath dir Github{..} = joinPath $
   dir : "defaults" : githubOwner : githubRepo : githubRef : githubPath
 
-data Result = Found | NotFound | Failed String
-  deriving (Eq, Show)
```

Comment:
(2) `Failed` is only ever used with `DownloadingFileFailed`; you could change that to `Failed URL Status` And convert to `DownloadingFileFailed` at the call site.

Reply:
That change has been made.